tests/stress_tests/scripts/run_memcheck.py

```diff
@@ -66,6 +68,29 @@ def json_load(path_or_string):
         return json.loads(path_or_string)
 
 
+def get_cli_arg(key: str, args: list):
+    """ Get value of command-line argument
+    """
+    if key[0] != "-":
+        key = "-" + key
```

Comment:
We shoudl not guess here. Let's just require full key name in a `key`. Finally in windows you may use / and in infrastructure there are key=value arguments without `-`

Reply:
Because implementation becomes difficult, I replaced `get_cli_arg` with additional ArgumentParser with `--test_conf` only